README.md

```diff
@@ -13,14 +13,11 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
   - [Install](#install)
   - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
```

Comment:
- [ ] As mentioned by the previous reviewer to make your README be more professional and descriptive, I think it would be good if you update some of the sub-sections in your `Getting Started` section with correct and appropriate information as the current ones don't give to the user(s) what he/they need to `clone` your application, `access` the cloned directory, `install bundles`, `Run your application`, `Run tests`,... To help you, I recommend you use **[this Gist file as example](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)** to update everything.

Reply:
Thanks so much. We will update it.
Thank you @tresorsawasawa